token/services/auditdb/db.go

```diff
@@ -253,15 +259,15 @@ func (d *DB) GetTokenRequest(txID string) ([]byte, error) {
 // AcquireLocks acquires locks for the passed anchor and enrollment ids.
 // This can be used to prevent concurrent read/write access to the audit records of the passed enrollment ids.
 func (d *DB) AcquireLocks(anchor string, eIDs ...string) error {
-	dedup := deduplicate(eIDs)
-	logger.Debugf("Acquire locks for [%s:%v] enrollment ids", anchor, dedup)
+	dedup := deduplicateAndSort(eIDs)
```

Comment:
You could use existing methods
```
dedup := collections.NewSet(eIDs...).ToSlice()
slices.Sort(dedup)
```

Reply:
Done. I have added also a comment to explain the problem we want to solve.